MyOwnORM/MyOwnORM/CustomKeyAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    public sealed class CustomKeyAttribute : Attribute
```

Comment:
Don't understand what type of key is it. If PK, rename maybe to 'CustomPrimaryKeyAttribute'

Reply:
Fixed